src/hevm/src/EVM/ABI.hs

```diff
@@ -134,6 +143,10 @@ data AbiType
   | AbiTupleType        (Vector AbiType)
   deriving (Read, Eq, Ord, Generic)
 
+instance ToJSON AbiType
```

Comment:
I think all these `ToJSON`/`FromJSON` instances are not neeeded anymore, they're just leftover from a time when I was serializing the corpus as json.